java/c/src/main/java/org/apache/arrow/c/Data.java

```diff
@@ -36,8 +36,7 @@
 /**
  * Functions for working with the C data interface.
  * <p>
- * This API is EXPERIMENTAL. Note that currently only 64bit systems are
- * supported.
```

Comment:
Do we want to remove this here? The fact that the Java API is experimental is a bit unrelated to the status of the C stream interface.
cc @zhztheplayer for opinions

Reply:
Yes afaik we don't yet have any codes about C Stream Interface in Java.